packages/react-core/src/modules/slack/hooks/useSlackChannels.ts

```diff
@@ -19,49 +21,66 @@ type UseSlackChannelOutput = {
   refetch: () => void;
 };
 
+type QueryKey = [key: string, cursor: string] | null;
+
+function getQueryKey(
```

Comment:
Nit: Should we move this generic query key logic to a swr utils file?

Reply:
In this case this is very specific to this slack channel payload so no on extracting right now.